package.json

```diff
@@ -29,9 +29,9 @@
   "license": "SEE LICENSE IN README.md",
   "dependencies": {
     "@babel/runtime": "^7.12.1",
+    "@devextreme/vdom": "^1.0.2",
```

Comment:
Do we use semver for new packages? If some breaking change is happened in generator, will version become 2.0.0?

Reply:
Yes, we want to avoid breaking changes in minor or patch. If we has some breaking cahanges, we will release a new major.
New features without BC we will release in minor, if we have fixes without features - it will be patch